integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -1008,6 +1062,46 @@ private static void verifyMemberHealth(String adminServerPodName, List<String> m
         });
   }
 
+  private void verifyWebLogicVersioninPod(int replicaCount) {
+    final String adminServerName = "admin-server";
+    final String managedServerNameBase = "ms-";
+    final String adminServerPodName = domainUid + "-" + adminServerName;
+    String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+
+    String introspectVersion =
+        assertDoesNotThrow(() -> getCurrentIntrospectVersion(domainUid, introDomainNamespace));
+
+    // verify admin server pods
+    logger.info("Verify weblogic.introspectVersion in admin server pod {0}", adminServerPodName);
+    verifyWebLogicVersionValue(adminServerPodName, introspectVersion);
+
+    // verify managed server pods
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Verify weblogic.introspectVersion in managed server pod {0}",
+          managedServerPodNamePrefix + i);
+      verifyWebLogicVersionValue(managedServerPodNamePrefix + i, introspectVersion);
+    }
+  }
+
+  private void verifyWebLogicVersionValue(String podName, String introspectVersion) {
```

Comment:
Change the method name to verifyIntrospectVersionLabelValue() to represent the action

Reply:
changed